src/main.cpp

```diff
@@ -67,6 +70,11 @@ int main(int argc, char *argv[])
         "AbstractLinkNamespace", 1, 0, "AbstractLinkNamespace", "This is another enum."
     );
 
+    qmlRegisterUncreatableMetaObject(
+        PingEnumNamespace::staticMetaObject,
+        "PingEnumNamespace", 1, 0, "PingEnumNamespace", "This is another enum."
```

Comment:
"this is another enum" is a bad description. This description is used thru IDE's to give us hints about what the code is.